tests/project/_common/test_movement.py

```diff
@@ -0,0 +1,437 @@
+            overwrite_existing=True,
+        )
+
+    # fail if not present
+    with pytest.raises(ValueError):
+        categorization.import_artifacts(
+            project_artifact_path=artifact_path,
+            tamr_client=client,
+            target_project=existing_project,
+            include_destructive_artifacts=[catfacts.CATEGORIZATION_TAXONOMIES],
+            fail_if_not_present=True,
+            overwrite_existing=True,
+        )
+
+    # test incorrect artifact name
+    with pytest.raises(ValueError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            include_additive_artifacts=["incorrect_artifact_name"],
+            overwrite_existing=True,
+        )
+    with pytest.raises(ValueError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            include_destructive_artifacts=["incorrect_artifact_name"],
+            overwrite_existing=True,
+        )
+    # Expected ValueError but got RuntimeError instead
+    with pytest.raises(RuntimeError):
+        categorization.import_artifacts(
+            tamr_client=existing_project.client,
+            project_artifact_path=artifact_path,
+            target_project=existing_project,
+            exclude_artifacts=["incorrect_artifact_name"],
+            overwrite_existing=True,
+        )
+
+
+@mock_api()
+def test_export():
+    client = utils.client.create(**CONFIG["toolbox_test_instance"])
+
+    project = client.projects.by_resource_id(CONFIG["projects"]["minimal_categorization"])
+    path_export_dir = "/home/ubuntu/tamr/projectExports"
+    exclude_list = [
+        catfacts.CATEGORIZATION_VERIFIED_LABELS,
+        catfacts.CATEGORIZATION_FEEDBACK,
+        catfacts.CATEGORIZATION_TAXONOMIES,
+    ]
+
+    op = categorization.export_artifacts(
+        project=project,
+        artifact_directory_path=path_export_dir,
+        exclude_artifacts=exclude_list,
+        asynchronous=False,
+    )
+
+    assert op.succeeded()
+
+
+@mock_api()
+def test_import_new():
+    client = utils.client.create(**CONFIG["toolbox_test_instance"])
+    project = client.projects.by_resource_id(CONFIG["projects"]["minimal_categorization"])
+
+    # export a project
+    op = categorization.export_artifacts(
+        project=project,
+        artifact_directory_path="/home/ubuntu/tamr/projectExports",
+        exclude_artifacts=None,
+        asynchronous=False,
+    )
+
+    assert op.succeeded()
+
+    # get artifact path
+    description = op.description
+    artifact_path = str(description).split(" ")[-1]
+
+    assert artifact_path[0] == "/"
+    assert artifact_path[-4:] == ".zip"
+
+    # import new project if it doesn't exist already
+    new_project_name = "new_categorization"
```

Comment:
Should you add a timestamp to your new_project_name to make it trully unique?

Reply:
same as above, I found it best to create unique pre-determined names without timestamps to get successful offline tests with the mock_api().